packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -290,9 +297,11 @@ export class AutoScalingGroup extends cdk.Construct implements cdk.ITaggable, el
   /**
    * Add command to the startup script of fleet instances.
    * The command must be in the scripting language supported by the fleet's OS (i.e. Linux/Windows).
+   *
+   * @deprecated Use userdata.addCommands() instead.
```

Comment:
What's wrong with this as sugar? Looks pretty useful...

Reply:
Maybe. I want to guide people towards the object that has convenience methods for common operations, so they don't go templating some complicated strings if there are better solutions available.